airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeCopyAzureBlobStorageDestination.java

```diff
@@ -63,6 +65,15 @@ public SqlOperations getSqlOperations() {
     return new SnowflakeSqlOperations();
   }
 
+  @Override
+  protected void performCreateInsertTestOnDestination(final String outputSchema,
+                                                      final JdbcDatabase database,
+                                                      final NamingConventionTransformer nameTransformer)
+      throws Exception {
+    AbstractJdbcDestination.attemptSQLCreateTableThenInsertDummyRecThenDropThisTableOperations(outputSchema, database, nameTransformer,
```

Comment:
this is a pretty gnarly name :) I'd rename it to `attemptTableOperations`

Reply:
Updated, thanks